dev/env/scripts/lib.sh

```diff
@@ -208,6 +210,7 @@ wait_for_resource_to_appear() {
 
     log "Waiting for ${kind}/${name} to be created in namespace ${namespace}"
 
+    # shellcheck disable=SC2034
     for i in $(seq "$seconds"); do
```

Comment:
Wouldn't this be properly fixed by
```suggestion
    for _ in $(seq "$seconds"); do
```